lib/flutter.yaml

```diff
@@ -0,0 +1,17 @@
+# Lints recommended for Flutter apps and packages.
+
+include: package:lints/recommend.yaml
```

Comment:
"recommended.yaml" now.

Can this be a relative path, so just "recommended.yaml"?

Reply:
Changed to the new name.

It appears that a relative path works here, I was just following how "recommended.yaml" is importing core:

https://github.com/dart-lang/lints/blob/1f80da49abf8b4ea15461e42ebb09eb51917b49d/lib/recommended.yaml#L1

@pq Should we change that and this one to use the relative path?

Reply:
cc @bwilkerson who knows more about nuances of `includes`...  Brian: any reason *not* to use a relative path import?